contrib/tig-completion.bash

```diff
@@ -33,8 +33,6 @@ __tig_options="
 	-v --version
 	-h --help
 	-C
-	--
```

Comment:
Hmm, why did you reve these two lines? They are valid completion options :)

Reply:
Because they confuse `__gitcomp`. Well, at least the `--` one.

Most commands do have the `--` option, but none that I'm aware of actually add it to the completion.

Reply:
okay, then that sounds fine to me too.